### ISSUE TITLE
Update to innonet 20

### ISSUE THREAD

**Issue by thomas-senechal** [massalabs/thyra#521]
**Context**
Testnet 20 got release a week ago and some partners need the innonet 20.

**How to**
Update Innonet address and DNS address

**Technical details**
Update Innonet address and DNS address in `int/api/server.go`

**QA testing**
1. Starts Thyra using `--node-server INNONET`
2. Upload a website
3. Navigate on some websites

**Comment by thomas-senechal** [comment]
I'm currently investigating a bug related to the event manager. 
It looks like the event manager never receives the events, and so, the API call never ends and the front end stepper never moves on.

**Comment by thomas-senechal** [comment]
> I'm currently investigating a bug related to the event manager. It looks like the event manager never receives the events, and so, the API call never ends and the front end stepper never moves on.

Found the issue.
In some cases, the event was emitted in the period prior to the one we were starting with (Massa is too fast 😎).
By using `get_filtered_sc_output_event` with a start period before the one we are in, we are able to catch the events emitted in the previous period. 

For flexibility, I added a `periodThreshold` and a `tickerDuration` constants to allow us to try some other things if the issue happens again in the future. 

IMO, we should dig into this issue and find the best way to fix it once the event manager is plugenized.
